package.json

```diff
@@ -8,6 +8,9 @@
     "build:dev": "npm run clean && webpack --config webpack/config.js --profile --json > stats.json",
     "build:dev:analyze": "npm run build:dev && npm run analyze",
     "build:dev:run": "npm run build:dev && node dist/index.js | pino-pretty -c -S",
+    "build:prod": "npm run clean && NODE_ENV=prod webpack --config webpack/config.js --profile --json > stats.json",
+    "build:prod:analyze": "npm run build:prod && npm run analyze",
+    "build:prod:run": "npm run build:prod && node dist/index.js | pino-pretty -c -S",
```

Comment:
`pino-pretty` [should not be used in production](https://github.com/pinojs/pino-pretty?tab=readme-ov-file#programmatic-integration), please remove the piping of the output to `pino-pretty`.

Reply:
Good call, I will fix.